rainloop/app.php

```diff
@@ -36,19 +36,28 @@
 		$sEncodedPassword = '';
 
 		$sUser = OCP\User::getUser();
+    $sPasswordSalt = '';
 
 		if (\OC::$server->getConfig()->getAppValue('rainloop', 'rainloop-autologin', false))
 		{
 			$sEmail = $sUser;
+      $sPasswordSalt = $sUser;
+			$sEncodedPassword = \OC::$server->getConfig()->getUserValue($sUser, 'rainloop', 'rainloop-autologin-password', '');
+		}
+		else if (\OC::$server->getConfig()->getAppValue('rainloop', 'rainloop-autologin-with-email', false))
+		{
+			$sEmail = \OC::$server->getConfig()->getUserValue($sUser, 'settings', 'email','');
+      $sPasswordSalt = $sUser;
 			$sEncodedPassword = \OC::$server->getConfig()->getUserValue($sUser, 'rainloop', 'rainloop-autologin-password', '');
 		}
 		else
 		{
 			$sEmail = \OC::$server->getConfig()->getUserValue($sUser, 'rainloop', 'rainloop-email', '');
+      $sPasswordSalt = $sEmail
```

Comment:
Here a semicolon is missing at the end of the line. That crashes the app right on start.